### DISCUSSION TITLE
How Can I Change My Computer Theme's Color To Get a Better Mica Effect?

### DISCUSSION THREAD

**Discussion by desmondalohan** [MicaForEveryone/MicaForEveryone#325] [Q&A]
I noticed that the Mica effect on the title bar is darker than the Mica effect on the window. I think this is related to how dark the theme is on the title bar compared to how dark the theme is on the window. The image attached will hopefully give you a better idea of what I mean:
![Screenshot 2023-07-12 115838](https://github.com/MicaForEveryone/MicaForEveryone/assets/82041382/f0b4ba08-db97-4e93-9f78-26cf18684a28)

**Comment by dongle-the-gadget** [comment]
It's the window (most likely your custom theme) using some kind of transparency masking. Nothing we can do here.

**Reply by desmondalohan** [reply]
Ok, thanks.

**Reply by musalahmu** [reply]
how did you make the (X) close button look like macos

**Reply by NANNO35578** [reply]
> how did you make the (X) close button look like macos

perhaps you can use this https://winmoes.com/tools/12948.html
